protocols/interfaces.go

```diff
@@ -33,6 +33,7 @@ type ChainTransaction struct {
 type SideEffects struct {
 	MessagesToSend       []Message
 	TransactionsToSubmit []ChainTransaction
+	ProposalsToProcess   []consensus_channel.Proposal
```

Comment:
Will this slice ever contain more than one element?

Reply:
If an intermediary is follower in two ledger channels it could contain a proposal from each side.